### ISSUE TITLE
targetdirectory: Relative Directory to Current File

### ISSUE THREAD

**Issue by goesredy** [codelios/vscode-dartsass#5]
My SCSS directory tree (example):
- assets/
   - css/
   - scss/

Is there a way to make the watcher put compiled css in css folder ( relative file_path instead relative to project folder ) ?
I don't think I can use directory relative to file_path, because only relative to project root is available.

**Comment by akkumar** [comment]
Sorry - at this moment - targetDirectory applies only relative to projectRoot. 

The interpretation becomes more hazy if targetDirectory changes for every file depending on the location. 

Can you specify `assets\css` as targetDirectory and would that help your case though ?

**Comment by goesredy** [comment]
@akkumar Yes, but if I work with multiple environment (eg. html and php app), this will need manual replace.
But really big thanks if this will available in the future.

**Comment by jeffglynn** [comment]
Sure would be nice to be able to specify relative output path to the current file

**Comment by developedsoftware** [comment]
We have this exact same requirement!
